### ISSUE TITLE
Dahua IP camera and nginx reverse proxy

### ISSUE THREAD

**Issue by marsonpl** [NginxProxyManager/nginx-proxy-manager#3771]
Hello

I have dahua IP camera and I want do access its web gui at cam.mydomain.com.

The web gui works without problem from LAN at internal IP on 192.168.1.108:8008 for now I opened the 8008 (web GUI) and 554 (RTSP) port to the outside world in my router and when I enter http://my.external.IP.address:8008 it also works however when I configure the reverse proxy host on nginx proxy manager I can login to the webui but when I click on live tab I see only white square, I assume that the issue might be an 554 port not being proxied through nginx proxy manager - any advice?

**Comment by marsonpl** [comment]
It seems that RTSP port is not needed, I closed RTSP port on router then I ask my colleague to test it, when he enter to cam.mydomain.com the video was not displayed but if he enter the IP:8008 directly the video was shown corectly, any idea why?

**Comment by marsonpl** [comment]
Yes - it is properly configured - othervise my colleague wont be able to connect to my IP camera via IP:port - he is in totally different network, I also tried it myself by using VPN and through http://ip:port it works OK when I am connected via VPN.

**Comment by clock145** [comment]
mark，遇到一样的问题，我抓包看，ABC三台机器，B没有回包给A。应该是udp包，抓包软件里面看是RTP协议。

**Comment by tristan-k** [comment]
Still needed. Also see this [issue](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/463).

**Comment by tristan-k** [comment]
Bump.
